browser/src/map/handler/Map.Keyboard.js

```diff
@@ -140,9 +140,110 @@ L.Map.Keyboard = L.Handler.extend({
 	},
 
 	keyCodes: {
+
+		BACKSPACE:8,
+		TAB:      9,
+		enter:   13, 
+		SPACE :   32,
 		pageUp:   33,
 		pageDown: 34,
-		enter:    13
+		SHIFT:    16, // shift		: UNKOWN
+		CTRL:     17, // ctrl		: UNKOWN
+		ALT:      18, // alt		: UNKOWN
+		pause:    19, // pause/break	: UNKOWN
+		capslock: 20, // caps lock	: UNKOWN, 
+		END:      35,
+		HOME:     36,
+		LEFT:     37,
+		UP:       38,
+		RIGHT:    39,
+		DOWN:     40,
+		INSERT:   45,
+		DELETE:   46,
+		
+		NUM0:     [48,96], // two values because of different mapping in mac and windows for same keys
+		NUM1:     [49,97],
+		NUM2:     [50,98],
+		NUM3:     [51,99],
+		NUM4:     [52,100],
+		NUM5:     [53,101],
+		NUM6:     [54,102],
+		NUM7:     [55,103],
+		NUM8:     [56,103],
+		NUM9:     [57,104],
+		A:        65,
+		B:        66,
+		C:        67,
+		c:        99,
+		D:        68,
+		E:        69,
+		//f:        70, no need for separate as for Windows it will remain the same
+		F:        70,
+		G:        71,
+		H:        72,
+		I:        73,
+		J:        74,
+		K:        75,
+		L:        76,
+		M:        77,
+		N:        78,
+		O:        79,
+		P:        80,	
+		Q:        81,
+		R:        82,
+		S:        83,
+		T:        84,
+		U:        85,
+		V:        86,
+		v:        118, // this is for MAC as the value of v changes when we press keydown
```

Comment:
I think we should unify that with the NUMX approach -> use an array with 2 values here V: [86, 118]

We can also add name for accessing those:
keyCodes.NUM3[DEFAULT] or keyCodes.NUM3[MAC]
where DEFAULT = 0, MAC = 1

Reply:
Done